crates/libs/wallet/src/types/compact_wad.rs

```diff
@@ -225,3 +228,282 @@ where
     let bytes = Vec::<u8>::deserialize(deserializer)?;
     PublicKey::from_slice(&bytes).map_err(serde::de::Error::custom)
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use nuts::nut00::secret::Secret;
+    use nuts::nut01::PublicKey;
+    use nuts::nut02::KeysetId;
+    use nuts::{Amount, traits::Unit};
+    use std::str::FromStr;
+
+    #[derive(Debug, Copy, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+    #[serde(rename_all = "lowercase")]
+    pub enum TestUnit {
+        Sat,
+    }
```

Comment:
there are already TestUnit available for you to use in crates/libs/nuts/src/lib.rs

Use it instead of recreating a new one with the whole implementation

Reply:
I told you in https://github.com/nutty-raccoon/paynet/pull/169/files#r2232352222